src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerEngineUnitTest.java

```diff
@@ -44,7 +45,15 @@ public void testIsActive() {
               final ReferenceDataSource ref = new ReferenceFileSource(reference) ) {
 
             final HaplotypeCallerEngine hcEngine = new HaplotypeCallerEngine(hcArgs, reads.getHeader(), reference.getAbsolutePath());
-            final Iterator<GATKRead> readIter = new ReadFilteringIterator(reads.query(paddedShardInterval), HaplotypeCallerEngine.makeStandardHCReadFilter(hcArgs, reads.getHeader()));
+
+            List<ReadFilter> hcFilters = HaplotypeCallerEngine.makeStandardHCReadFilters();
+            hcFilters.forEach(filter -> filter.setHeader(reads.getHeader()));
+            ReadFilter hcCombinedFilter = hcFilters.get(0);
+            for ( int i = 1; i < hcFilters.size(); ++i ) {
+                hcCombinedFilter = hcCombinedFilter.and(hcFilters.get(i));
```

Comment:
Yes, that's a good idea -- created https://github.com/broadinstitute/gatk/issues/2198 for this task.